### DISCUSSION TITLE
Votes on a new domain name

### DISCUSSION THREAD

**Discussion by asim** [micro/go-micro#2068] [Q&A] [answered]
Go Micro is now Nitro. It's new domain is go-nitro.dev but we can do way better than that. Give us some ideas for a new domain.

gonitro.no or .dev
getnitro.no or .dev

More suggestions welcome.

**Comment by starfork** [comment]
gonitro.dev

**Comment by 38b3eff8** [comment]
gonitro.dev

**Comment by asim** [comment]
Domain gonitro.dev purchased. Changing soon.

**Comment by asim** [comment, marked as answer]
Now live

**Comment by asim** [comment]
Next step, proper webpage
